docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -100,10 +115,22 @@ In the folder structure illustrated below, the top-level directory is *flower_ph
             10791227_7168491604.jpg
 ```
 
+## Set up object detection image data files
+
+Model Builder expects object detection image data to be in JSON format generated from [VoTT](https://github.com/Microsoft/VoTT/releases). The JSON file is located in the **vott-json-export** folder in the **Target Location** that is specified in the project settings.
+
+The JSON file consists of the following information generated from VoTT:
+
+- All tags that were created
+- The image file locations
+- The image bounding box information
+- The tag associated with the image
+
```

Comment:
Link to this section here too.

"For more information on preparing data for object detection, see [label images for object detection using VoTT](./label-images-for-object-detection-using-vott.md)